bool found
string name
string label
int32 tag_id
string frame_id
time last_seen
int32 times_seen
geometry_msgs/PoseStamped pose
